### ISSUE TITLE
Is it support AMD Ryzen CPU's?

### ISSUE THREAD

**Issue by allakov** [yangxiaohua2009/custom-kernel#13]
Very appreciated that you maked a fix for Linux Kernel with ESSX8336 Soundcard problem.
But I'm using Huawei Mate D16 notebook with AMD Ryzen processor and I noticed that fix for Intel CPU's only? If you can advice me, is there any way to run it on AMD?

**Comment by yangxiaohua2009** [comment]
We have a machine driver for [stoney platform.](https://www.mail-archive.com/dri-devel@lists.freedesktop.org/msg403025.html)
There is no machine driver for Ryzen CPU yet.
You can try AMD HDA HDMI driver to see if you can get sound from HDMI.

**Comment by andreaconsole** [comment]
Hi everyone, I'm using Kubuntu 23.04 kernel 6.2.0-22. Still no sound... Any hope to have this fixed soon? thanks.

**Comment by yangxiaohua2009** [comment]
You may want to try these patches to enable ES8336 sound on AMD platform

https://www.spinics.net/lists/alsa-devel/msg155989.html

**Comment by andreaconsole** [comment]
Hi @yangxiaohua2009, thanks for your quick reply. I apologise for my poor understanding on patches life cycle, but it looks like they have been proposed for merging into the current development of the Linux kernel (since Mark Brown is involved). Does it mean that they eventually might get to one of the next official kernels? Applying patches and compile kernels myself is a little out of my league...

**Comment by yangxiaohua2009** [comment]
@andreaconsole No, the patch has not been approved. The patch is workable though. Maybe you should email the maintainer and ask if he can provide you with the compiled deb kernel?

**Comment by andreaconsole** [comment]
Thanks for the clarification, @yangxiaohua2009 . Yes, this is an option I can explore. Thanks!

**Comment by andrey-voter** [comment]
> Thanks for the clarification, @yangxiaohua2009 . Yes, this is an option I can explore. Thanks!

Hello! I'm using Huawei matebook D16 and have same problem. Did you find a solution? Thanks
